tests/Unit/Lib/RouterTest.php

```diff
@@ -138,6 +138,31 @@ public function testIsCallable()
         $this->assertFalse(Router::isCallable($controller, 'preAction'));
         $this->assertFalse(Router::isCallable($controller, 'nonexistentMethod'));
         $this->assertFalse(Router::isCallable(null, null));
+
+        #
+        # TODO: add tests for __call
+        # NOTE: phpunit currently cannot handle mocking __call properly
+        # see https://github.com/sebastianbergmann/phpunit-mock-objects/issues/275
+        #
+        /*
+        $magicController = $this->getMockBuilder('Controller')
```

Comment:
Why not create a test controller that uses magic methods and assert it works as example expected?